.github/workflows/mupdf.yml

```diff
@@ -2,8 +2,8 @@ on:
   workflow_dispatch:
   push:
     branches:
-      - '**'
-      - '!main'
+      - "**"
+      - "!main"
     paths:
       - misc/mupdf/version
       - .github/workflows/mupdf.yml
```

Comment:
can you double check if the change in binaries is valid @diegobernardes . Not sure why the workflow file is in the `paths`